ui/Makefile

```diff
@@ -49,6 +51,10 @@ $(TYPESCRIPT_TARGET): $(shell find $(TS_ROOT))
 	$(NODE_MODULES)/.bin/tsc -p $(TS_ROOT)
 	$(NODE_MODULES)/.bin/tslint -c $(TS_ROOT)/tslint.json $(addprefix -f , $(shell find $(TS_ROOT) -name *.ts -not -path *typings*))
 
+$(STYLUS_TARGET): css/*.styl
+	npm install
+	$(NODE_MODULES)/.bin/stylus $(CSS_ROOT)
```

Comment:
Although: https://github.com/github/linguist/blob/master/lib/linguist/generated.rb#L92

So either minify or just add a bunch of long line trash?